templates/undercloud-networks-routed.yaml

```diff
@@ -11,6 +11,11 @@ parameters:
     description: CIDR for provision network subnet
     default: 192.0.2.0/24
 
+  provision_net_allocation_pools:
+    type: json
+    description: Allocation pool for third provision network subnet
+    default: [{"start": "192.0.2.200", "end": "192.0.2.254"}]
```

Comment:
Agreed, these are internet routeble addresses so one day we will fail to access something out there because of this. Let's use 192.168.[24, 25, 26].0/24 ?

Reply:
Sounds good.

Reply:
Done, hope I found and updated all 192.0.2.x entries.
commit: a16f379e1bb8e9bb52db1cc828a55add8d3c138f